bambam.py

```diff
@@ -23,247 +23,244 @@
-    col.hsva = (hue, 100, 100, 50)
-
-    return Color(col.r, col.g, col.b)
-
-
-# Load image/, handling setting of the transparency color key
-def load_image(fullname, colorkey = None):
-    try:
-        image = pygame.image.load(fullname)
-    except pygame.error as message:
-        print("Cannot load image:", fullname)
-        raise SystemExit(message)
-    image = image.convert()
-    if colorkey is not None:
-        if colorkey is -1:
-            colorkey = image.get_at((0, 0))
-        image.set_colorkey(colorkey, RLEACCEL)
-    return image
-
-
-# Load sound file in data/
-def load_sound(name):
-    class NoneSound:
-        def play(self): pass
-    if not pygame.mixer:
-        return NoneSound()
-    try:
-        sound = pygame.mixer.Sound(name)
-    except pygame.error as message:
-        print("Cannot load sound:", name)
-        raise SystemExit(message)
-    return sound
-
-
-def load_items(lst, blacklist, load_function):
-    """Runs load_function on elements of lst unless they are blacklisted."""
-    result = []
-    for name in lst:
-        if True in [fnmatch.fnmatch(name, p) for p in blacklist]:
-            print("Skipping blacklisted item:", name)
-        else:
-            result.append(load_function(name))
-    return result
-
-
-# Processes events
-def input(events, quit_pos):
-    global sequence, mouse_down, sound_muted
-    for event in events:
-        if event.type == QUIT:
-            sys.exit(0)
-
-        # handle keydown event
-        elif event.type == KEYDOWN or event.type == pygame.JOYBUTTONDOWN:
-            # check for words like quit
-            if event.type == KEYDOWN:
-                if is_latin(event.key):
-                    sequence += chr(event.key)
-                    if sequence.find('quit') > -1:
-                        sys.exit(0)
-                    elif sequence.find('unmute') > -1:
-                        sound_muted = False
-                        #pygame.mixer.unpause()
-                        sequence = ''
-                    elif sequence.find('mute') > -1:
-                        sound_muted = True
-                        pygame.mixer.fadeout(1000)
-                        sequence = ''
-
-            # Clear the background 10% of the time
-            if random.randint(0, 10) == 1:
-                screen.blit(background, (0, 0))
+class Bambam:
+    args = None
+    progInstallBase = None
+
+    colors = None
+    images = None
+    sounds = None
+
+    background = None
+    screen = None
+    sheight = None
+    swidth = None
+
+    mouse_down = None
+    sequence = None
+    sound_muted = None
+
+    @classmethod
+    def get_color(cls):
+        ''' Return bright color varying over time '''
+        col = Color('white');
+
+        hue = pygame.time.get_ticks() / 50 % 360
+        col.hsva = (hue, 100, 100, 50)
+
+        return Color(col.r, col.g, col.b)
+
+    @classmethod
+    def load_image(cls, fullname, colorkey = None):
```

Comment:
Please no spaces around `=` for default kwarg values.

Reply:
I went a bit beyond here, and just ran `autopep8` on the source, then did some manual fixup based on `flake8`. I purposefully left a few lines beyond 79 characters, because the fix was nonsensical.

Reply:
Note: I see there are a couple of other pull requests pending. The linting will probably make then conflict if you merge this.

In my experience, though, just running `autopep8` on the proposed changes should go most of the way, and the rest of the conflicts should be relatively trivial to fix.